src/backend/commands/session/join_session.cpp

```diff
@@ -50,6 +50,21 @@ namespace big
 			return valid_args;
 		}
 
+		virtual std::optional<std::vector<std::string>> get_argument_suggestions(int arg) override
+		{
+			if (arg == 1)
+			{
+				std::vector<std::string> suggestions;
+				for (const auto& [_, session_type_string] : m_session_types)
```

Comment:
If you're not making use of the key then consider reworking the for loop to make use of `std::ranges::views::values`.

```cpp
for (const auto& session_type_string : m_session_types | std::ranges::views::values)
{

}
```